examples/service-alias/service-alias.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.8;
+
+import "suave-std/suavelib/Suave.sol";
+import "suave-std/Suapp.sol";
+
+contract ServiceAlias is Suapp {
+    function exampleCallback() public {}
+
+    function example() public returns (bytes memory) {
+        Suave.HttpRequest memory request;
+        request.url = "example";
+        request.method = "GET";
+        request.timeout = 1000;
+
+        bytes memory response1 = Suave.doHTTPRequest(request);
+
+        // Make the request to the http endpoint
+        request.url = "http://example.com";
+        bytes memory response2 = Suave.doHTTPRequest(request);
```

Comment:
Why is this http, but when we start Suave locally it is HTTPS?

Reply:
Fixed